.eslintrc.cjs

```diff
@@ -3,9 +3,13 @@ module.exports = {
   env: {
     node: true
   },
+  plugins: ["jsdoc"],
   extends: [
     'digitalbazaar',
     'digitalbazaar/jsdoc',
     'digitalbazaar/module'
-  ]
+  ],
+  rules: {
+    "jsdoc/newline-after-description": 0,
```

Comment:
What did you add this for? Was it not linting properly before?

Reply:
Its because its a new linting validation, I just removed it because I don't think we care